e2e/core/user-info-component-cloud.e2e.ts

```diff
@@ -29,7 +29,7 @@ describe('User Info - SSO', () => {
     const loginSSOPage = new LoginSSOPage();
     const navigationBarPage = new NavigationBarPage();
     const userInfoPage = new UserInfoPage();
-    const identityService: Identity = new Identity();
+    const identityService: Identity = new Identity(undefined);
```

Comment:
Shouldn't we have new Identity() instead of new Identity(undefined)?

Reply:
undefined is added for the tests to compile. as the constructor expects a parameter here.

Reply:
Maybe we can have also a constructor without params?

Reply:
will check.